draft-ietf-opsawg-teas-attachment-circuit.md

```diff
@@ -1250,59 +1250,54 @@ The attachment circuit in this case use a SAP identifier to refer to the physica
+For readability the payload is displayed as single JSON file ({{parent-profile}}). In practice, several API calls make take place to initialize these resources (e.g., GET requests from the customer to retrieve the IP address pools for SFs on "vlan 100" thanks to parent configuration and BGP configuration, and POST extra routes for user planes and BFD).
 
-~~~~ json
-{::include-fold ./json-examples/svc/ac-cloud-bfd.json}
-~~~~
-{: #get-bfd-profiles title="Message Body of a Response to get available BFD profiles from Provider Network"}
+Note that no individual IP address is assigned in the data model for the SF user plane instances (i.e., no "customer-address" in the Child AC). The assignment of IP addresses to the SF endpoints is managed by the Cloud Infrastructure IPAM based on the customer-addresses IP address pool "192.0.2.1-200". Like in any standard LAN-facing scenario, it is assumed that the actual binding of IP endpoints to logical attachments (here Child ACs) relies on a dedicated protocol logic  (typically, ARP or NDP) and is not captured in the data model. Hence, the IP addresses displayed for SF user plane instances are simply examples of a realization approach. Note also that the Control Plane is defined with static IP address assignment on a given AC/bearer to illustrate another deployment alternative.
 
 ~~~~ json
 {::include-fold ./json-examples/svc/ac-cloud-parent.json}
 ~~~~
-{: #parent-profile title="Message body for configuration of the NF ACs"}
+{: #parent-profile title="Message Body for the Configuration of The SF ACs"}
 
-Assuming a failure of compute-01, the instance nf-up-1 is redeployed to compute-07 by the NF/Cloud Orchestration. Additionally, the nf can be scaled-out thanks to the creation of an extra instance nf-up7 on compute-08. Since networking is pre-provisionned, these operations happen without any API call. In other words, this is transparent from the perspective of the configuration of the provider network.
+Assuming a failure of "compute-01", the instance "nf-up-1" can be redeployed to "compute-07" by the SF/Cloud Orchestration. Additionally, the SF can be scaled-out thanks to the creation of an extra instance "nf-up7" on "compute-08". Since connectivity is pre-provisioned, these operations happen without any API calls. In other words, this redeployment is transparent from the perspective of the configuration of the provider network.
 
 ~~~~ aasvg
 {::include-fold ./figures/ac-parent-nf-lcm.txt}
```

Comment:
What does it means "DOWN" in the figure?

Reply:
it is broken (failure ?)

Reply:
I still really  hate SF - outside classics such as Dune, Old Star Wars :-)  - 
Afaik, NF is used in IETF: https://datatracker.ietf.org/doc/html/draft-ietf-teas-5g-ns-ip-mpls

Reply:
;-) 

I have a consistency concern here. Using SF makes my life easier. This is an appendix after all. I will see how to reconcile all views here.